proposals/NNNN-New_rules_for_providing_VRHelpItems_VRHelpTitle.md

```diff
@@ -0,0 +1,68 @@
+```
+# Feature name
+
+* Proposal: [SDL-NNNN](NNNN-New_rules_for_providing_VRHelpItems_VRHelpTitle.md)
+* Author: [Irina Getmanets](https://github.com/GetmanetsIrina)
+* Status: **Awaiting review**
+* Impacted Platforms: Core, RPC
+
+## Introduction
+
+VRHelp items and VRHelpTitle are set by SetGlobalProperties for helping user to make voice navigation over registered application, what is very usefull in case your hands are busy with driving.
+
+## Motivation
+
+If there is no SetGlobalProperties issued, the default help item list will be populated by command list shared by Voice module or by default values from .ini file. If App issues a SetGlobalProperties with help item list, then this list will have precedence and it will be displayed on HMI.
+We imagine cases when app does not provide VRHelps item and VRHelpTitle, so in such case help information will be empty.
```

Comment:
@GetmanetsIrina ,

Instead of "We imagine cases when app does not provide VRHelps item and VRHelpTitle"
better to write
"But there is no guarantee that Application provide VRHelps and VRHelpTitle, and possible situation when driver couldn't use Voice Recognition"

Reply:
Changed locally.